src/components/Common/TaxInputs/TaxInputs.tsx

```diff
@@ -66,9 +69,11 @@ export function NumberInput({
   const { key, label, description } = question ? question : requirement
   const value = question ? question.answers[0]?.value : requirement.value
 
+  if (!key) return null
```

Comment:
If there's no key, this won't get associated with a hook form input and won't actually submit anything so no sense in rendering an input in that case